src/main/java/com/redhat/red/build/koji/model/json/EmptyTypeInfoExtraInfo.java

```diff
@@ -21,26 +21,26 @@
  * Created by pkocandr on 06/22/21
  */
 @StructPart
-public class NpmTypeInfoExtraInfo
+public class EmptyTypeInfoExtraInfo
```

Comment:
I think the fact that it is empty is just incidental. I'd rather see this class and the type information kept by just making another copy of this class.

Related to this, we have a potential problem deserializing something that has (in Python) `{ "npm" : None }`. This will result in a `null` field, which is the same as the key not existing, isn't it? We have to look at how we're deserializing to make sure we're not losing the type information.

Reply:
I don't think it is incidental as it was purposely added by @pkocandr in https://github.com/release-engineering/kojiji/pull/144 ; I have just renamed and reused it here. *If* this messes up the encoding then sure, I can roll it back

Reply:
Note : need to establish from Koji / Konflux / etc teams what are the accepted forms for rpm/npm/etc.